Makefile

```diff
@@ -14,11 +14,11 @@ build:
 
 .PHONY: run
 run:
-	docker run --rm --name jaeger -e JAEGER_DISABLED=false --link some-clickhouse-server -it -u ${shell id -u} -p16686:16686 -p14250:14250 -p14268:14268 -p6831:6831/udp -v "${PWD}:/data" -e SPAN_STORAGE_TYPE=grpc-plugin jaegertracing/all-in-one:${JAEGER_VERSION} --query.ui-config=/data/jaeger-ui.json --grpc-storage-plugin.binary=/data/jaeger-clickhouse-$(GOOS)-$(GOARCH) --grpc-storage-plugin.configuration-file=/data/config.yaml --grpc-storage-plugin.log-level=debug
+	docker run --rm --name jaeger -e JAEGER_DISABLED=true -it -u ${shell id -u} --network="host" -p16686:16686 -p14250:14250 -p14268:14268 -p6831:6831/udp -v "${PWD}:/data" -e SPAN_STORAGE_TYPE=grpc-plugin jaegertracing/all-in-one:${JAEGER_VERSION} --query.ui-config=/data/jaeger-ui.json --grpc-storage-plugin.binary=/data/jaeger-clickhouse-$(GOOS)-$(GOARCH) --grpc-storage-plugin.configuration-file=/data/config.yaml --grpc-storage-plugin.log-level=debug
 
 .PHONY: run-hotrod
 run-hotrod:
-	docker run --rm  --link jaeger --env JAEGER_AGENT_HOST=jaeger  --env JAEGER_AGENT_PORT=6831  -p8080:8080 jaegertracing/example-hotrod:${JAEGER_VERSION} all
```

Comment:
May i ask why we decided to change from docker networking to host networking  ?

Reply:
Also why do we publish ports when using host networking ? 

```
WARNING: Published ports are discarded when using host network mode
```

Reply:
It breaks setup on MacOS for me, since in MacOS docker runs inside a linux VM.

Reply:
i used it to make run-hotrod compatible with using external storages, for wich I used "host" networking in run. But, I guess, it's better to get rid of "host" in Makefile.

Reply:
I made new pull request bringing old makefile schemes back.